snowpack/src/util.ts

```diff
@@ -3,16 +3,16 @@ import etag from 'etag';
 import execa from 'execa';
 import findUp from 'find-up';
 import fs from 'fs';
-import {isBinaryFile} from 'isbinaryfile';
+import { isBinaryFile } from 'isbinaryfile';
```

Comment:
run `yarn format` in the top-level workspace to re-format the monorepo and get rid of some of these style diffs